### DISCUSSION TITLE
Nameplate Support

### DISCUSSION THREAD

**Discussion by AlternativeSoap** [Zrips/CMI#6069] [Enhancement]
Is there a way to configure nameplate with gradient? I know I can use the command /nameplate but that doesn't work proberly with colors. I want it to show prefix and name with colors. Is it possible to add that to CMI.
And not the command but permission based nameplate over head with prefix and color.

Example:
NameplateGroups:
Default: {prefix}{displayname}{suffix}
Member: {prefix}{displayname}{suffix

**Comment by dan28000** [comment]
Did you try for example {#ff0000>}your text{#00ff00<}
In /nameplate %playername% pref: {#ff0000>}your text{#00ff00<}

**Reply by AlternativeSoap** [reply]
I know that. But is it possible to make it permission group based?

**Reply by AlternativeSoap** [reply]
If there is a way to do so tell me!
I can remove a whole plugin for that one thing

**Reply by ItLucifer-MRPG** [reply]
what do you mean permission group based?

**Reply by AlternativeSoap** [reply]
Look at "NameTagEdit" like that. 
It automatically changes nameplate over playerheads. 

When doing this example:
Group:
  Member: '&a'
  Admin: '&4'


Member will automatically change to green.
Admin to red.


Then I will insert example:

CMI.nameplate.member to member group in my permission group file "luckperm" in this case.


Nametagedit doesn't support hex. But cmi does.

**Reply by ItLucifer-MRPG** [reply]
I assume we are talking about /cmi nameplate (over head names rather than chat) 

Which you cant do it as if it was chat format by permissions. But however you can set up event commands to do it for you and make it something like:

Check:%luckperm_in_group_[groupName]%==[groupName]! cmi nameplate [playerName] -pref:{#Color>}[prefixYouWant]{#otherColor<}

And just set that for when they join the server so it always checks then

**Reply by AlternativeSoap** [reply]
Do I put anything inside "[playerName]"?

Check:%luckperm_in_group_crafter%==crafter! cmi nameplate [playerName] -pref:{#0a810a>}Crafter{#ffffff<}

So should it look like this

**Reply by ItLucifer-MRPG** [reply]
yep, [playerName] is a placeholder that cmi works with it will do it as the player that joined

**Reply by AlternativeSoap** [reply]
I'll test it

**Reply by ItLucifer-MRPG** [reply]
If you want help with things, Id suggest joining the discord, Its alot easier to get quick responses of ways to execute ideas on there

**Reply by AlternativeSoap** [reply]
Doesn't work.

**Reply by AlternativeSoap** [reply]
I will join.
